index.html

```diff
@@ -14,32 +14,48 @@ <h1> INTENTION TIMER </h1>
-        <section class="right-section">
-          <h2 class="past-activites">Past Activities</h2>
-          <aside id="asideContainer">
-            <p id="line1">You haven't logged any activities yet.</p>
-            <p id="line2">Complete the form on the left to get started!</p>
-          </aside>
-        </section>
+      <h2 class="new-activity">New Activity</h2>
+      <section class="activity-form">
+        <p class="select-category">Select a category:</p>
+        <button class="study-button" id="studyButton" type="button" name="button">Study</button>
+        <button class="meditate-button" id="meditateButton" type="button" name="button">Meditate</button>
+        <button class="exercise-button" id="exerciseButton" type="button" name="button">Exercise</button>
+        <p class="error-message hidden"> <img class="warning-img" src="./assets/warning.svg" alt="">Please choose a category.</p>
+        <form class="input-form" id="inputForm">
+          <p class="accomplish-text"> What would you like to accomplish during this time?</p>
+          <input class="accomplish-input" type="text">
+          <p class="error-message hidden"> <img class="warning-img" src="./assets/warning.svg" alt="">A description is required.</p>
+          <article class="time-container">
+            <p class="time-text">Minutes</p>
+            <input class="minutes-input" id="minutesInput" type="number">
+            <p class="error-message hidden"> <img class="warning-img" src="./assets/warning.svg" alt="">Minutes are required.</p>
+            <p class="time-text">Seconds</p>
+            <p class="error-message hidden"> <img class="warning-img" src="./assets/warning.svg" alt="">Seconds are required.</p>
+          </article>
+        </form>
+        <button class="start-activity" id="startActivity" type="button">START ACTIVITY</button>
+      </section>
+      <section class="timer-screen hidden-right-section">
+        <p id="user-accomplish-input"></p>
+        <p id="countdown-timer"><span id="minutes-display"></span>:<span id="seconds-display"></span></p>
+        <p id="finished-message" class="hidden-right-section">Great job, you have finished your activity!</p>
+        <button type="button" class="disable" id="start-button">START</button>
+        <button type="button" id="log-activity-button" class="hidden-right-section">LOG ACTIVITY</button>
+      </section>
+      <section class="create-new-activity-screen hidden-right-section">
+        <button type="button" class="create-new-activity-button">CREATE A NEW ACTIVITY</button>
+      </section>
+    </section>
+    <section class="right-section">
+      <h2 class="past-activites">Past Activities</h2>
+      <aside id="asideContainer">
+        <p id="line1">You haven't logged any activities yet.</p>
+        <p id="line2">Complete the form on the left to get started!</p>
+      </aside>
+    </section>
   </main>
-  <script src="./main.js"></script>
 </body>
+<script src="./main.js"></script>
```

Comment:
Not sure it matters, but I saw that you should add things that aren't main, above main?